src/sagas/pushNotification.js

```diff
@@ -511,6 +512,11 @@ export const getTxDetails = async (wallet, txId) => {
  */
 export function* loadTxDetails(action) {
   const { txId } = action.payload;
+  const isLocked = select((state) => state.lockScreen);
+  if (isLocked) {
+    yield take(isUnlockScreen);
```

Comment:
This will be stuck here until `isUnlockScreen` action is dispatched, right?

I guess that if the user resets the screen from the `LockScreen`, starts a new wallet, and then unlocks it, it will display the tx details screen, right?

We can handle that by using race, something like:

```javascript
const { resetWallet, unlockWallet } = yield race({
  unlockWallet: isUnlockScreen,
  resetWallet: <reset_wallet_action_here>,
});

if (resetWallet) {
  return;
}
```

Reply:
Resolvido em [591e6e3](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/261/commits/591e6e31dea4b71c68501a173e4430b3f1cea2ae) [bcf38ab](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/261/commits/bcf38abad7e148d55f126ff97571a3cbcc4ccebd)